min/lib/Minify.php

```diff
@@ -517,7 +518,7 @@ protected static function _combineMinify()
                 // minify previous sources with last settings
                 $imploded = implode($implodeSeparator, $groupToProcessTogether);
                 $groupToProcessTogether = array();
-                if ($lastMinifier) {
+                if ($lastMinifier && !$no_minify) {
```

Comment:
For JS, set `$source->minifier = ""`. For CSS, `$source->minifyOptions['compress'] = false;`